ui/next/config.js

```diff
@@ -137,6 +139,9 @@ System.config({
     "github:jspm/nodelibs-zlib@0.1.0": {
       "browserify-zlib": "npm:browserify-zlib@0.1.4"
     },
+    "npm:ByteBuffer@0.2.6": {
```

Comment:
I ran ```jspm install npm:ByteBuffer``` in order to get `ByteBuffer`s working in the browser. I'll take another look to see what's necessary.